github-electron/github-electron.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Electron v1.2.7
+// Type definitions for Electron v1.2.8
```

Comment:
Electron v1.3.0 was released without any change to APIs so should we mark as v1.3.0 here?